### PR TITLE
LPS-113126 Missing tags in New Question

### PR DESCRIPTION
**Pull request by LuismiBarcos**

### CODE REVIEW

**Comment by LuismiBarcos** [comment]
@nhpatt

File: modules/apps/questions/questions-web/src/main/java/com/liferay/questions/web/internal/portlet/QuestionsPortlet.java [hunk 1]
```diff
@@ -115,6 +124,34 @@ protected void setItemSelector(ItemSelector itemSelector) {
...
+			portletURLWrapper.setParameter(
+				"eventName", renderResponse.getNamespace() + "selectTag");
+			portletURLWrapper.setParameter(
+				"selectedTagNames", "{selectedTagNames}");
+			portletURLWrapper.setWindowState(LiferayWindowState.POP_UP);
+
+			return portletURLWrapper.toString();
+		}
+		catch (Exception exception) {
+			return null;
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/java/com/liferay/questions/web/internal/portlet/QuestionsPortlet.java:151]: relaunch exception

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/TagSelector.es.js [hunk 2]
```diff
@@ -71,17 +50,18 @@ export default ({tagsChange, tagsLoaded, tags = [], ...props}) => {
...
-					inputValue={inputValue}
-					items={items}
-					onChange={setInputValue}
-					onItemsChange={filterItems}
-					sourceItems={sourceItems}
-				/>
+				<div className="questions-tag-selector">
+					<AssetTagsSelector
+						eventName={`_${context.portletNamespace}_selectTag`}
+						groupIds={[context.siteKey]}
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/TagSelector.es.js:56]: siteKey works? I thought it needed the siteId

Comment (LuismiBarcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/TagSelector.es.js:56]: You are right but `context.siteKey` contains the ScopeGroupId. Maybe we should rename `siteKey` to siteId? We should use siteKey in the rest of Questions?

Comment (nhpatt) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/TagSelector.es.js:56]: Got it!

**Comment by nhpatt** [comment]
ci:forward

**Comment by nhpatt** [comment]
ci:forward

**Comment by nhpatt** [comment]
Rebased here: https://github.com/liferay-headless/liferay-portal/pull/61

**Pull request closed without merging**